core/templates/pages/topic-viewer-page/practice-tab/practice-tab.directive.ts

```diff
@@ -43,12 +43,43 @@ angular.module('oppia').directive('practiceTab', [
           var ctrl = this;
 
           ctrl.newPracticeSession = function() {
+            var commaSeparatedSelectedSubtopics = '';
+            for (var idx in ctrl.selectedSubtopicIndices) {
+              if (ctrl.selectedSubtopicIndices[idx]) {
+                commaSeparatedSelectedSubtopics += (
+                  ctrl.availableSubtopics[idx].getTitle() + ',');
+              }
+            }
             var practiceSessionsUrl = UrlInterpolationService.interpolateUrl(
               PRACTICE_SESSIONS_URL, {
-                topic_name: ctrl.getTopicName()
+                topic_name: ctrl.getTopicName(),
+                // These are actually the selected subtopics, but named so
+                // because this will be shown in the URL and for the learner,
+                // we surface the subtopics as 'Skills'.
+                selected_skills: commaSeparatedSelectedSubtopics
               });
             $window.location.href = practiceSessionsUrl;
           };
+
+          ctrl.disableStartButton = function() {
```

Comment:
I think this should be isStartButtonDisabled() or shouldStartButtonBeDisabled(), since it returns a boolean rather than performs an action.

Reply:
Done